src/routes/customerRoutes.js

```diff
@@ -5,16 +5,21 @@ const Validate = require("../middlewares/validate")
 const validator = require("../validators/vendorValidator")
 
 router.get(
-    "/vendors",
+    "/:customerId/vendors",
```

Comment:
@lilstex can you explain the reason for designing this route like this? I imagine it's for use by customers to list vendors, if so, what's the purpose of the `:customerId` as a route parameter?